water-sa.csl

```diff
@@ -0,0 +1,681 @@
+            <!--<text value="b.m." text-case="capitalize-first"/>-->
+          <!--</else>-->
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <group delimiter=" ">
+      <text variable="volume" font-weight="bold"/>
+      <text variable="issue" prefix="(" suffix=")"/>
+      <text macro="collection" prefix="(" suffix=")"/>
+      <text variable="page"/>
+    </group>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if type="broadcast motion_picture song report" match="any">
+        <choose>
+          <if variable="publisher">
+            <text variable="publisher"/>
+          </if>
+        </choose>
+      </if>
+      <else>
+        <choose>
+          <if variable="publisher">
+            <text variable="publisher"/>
+          </if>
+          <!--<else>-->
+            <!-- sine nomine (s.n.)-->
+            <!--<text value="b.n."/>-->
+          <!--</else>-->
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher-info">
+    <choose>
+      <if variable="publisher">
+        <group delimiter=", ">
+          <text macro="publisher"/>
+          <text macro="publisher-place"/>
+        </group>
+      </if>
+      <else>
+        <text macro="publisher-place"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="accessed">
+    <choose>
+      <if variable="URL DOI" match="any">
+        <group prefix=" (" suffix=")">
+          <text term="accessed" text-case="capitalize-first" suffix="&#160;"/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric"/>
+            <date-part name="month" prefix=" "/>
+            <date-part name="year" prefix=" "/>
+          </date>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <choose>
+      <if type="report">
+        <text variable="collection-title" font-style="italic"/>
+      </if>
+      <else>
+        <group delimiter=", ">
+          <text variable="collection-title"/>
+          <text variable="collection-number"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="page">
+    <choose>
+      <if type="book thesis manuscript" match="any">
+        <group delimiter="&#160;">
+          <text variable="number-of-pages"/>
+          <label variable="number-of-pages" form="short"/>
+        </group>
+      </if>
+      <else-if type="chapter paper-conference article-newspaper" match="any">
+        <group delimiter="&#160;">
+          <!--<label variable="page" form="short"/>-->
+          <text variable="page"/>
+        </group>
+      </else-if>
+      <else-if type="report patent" match="any">
+        <text variable="page" suffix=" "/>
+      </else-if>
+    </choose>
+  </macro>
+  <!--<macro name="issn">
+    <text variable="ISSN" prefix="ISSN&#160;"/>
+  </macro>-->
+  <macro name="url">
+    <choose>
+      <if variable="DOI">
+        <text variable="DOI" prefix="http://doi.org/"/>
```

Comment:
@libora, we standardized all styles to "https://doi.org/" for DOI URLs, even if the style guide uses HTTP or "dx.doi.org", since the former is now the officially preferred format.